docker-local/docker-compose.yml

```diff
@@ -325,6 +329,7 @@ services:
     volumes:
       - ./docker/wait4:/opt/auth-service/wait4
       - ./docker/auth-service/default.json:/opt/auth-service/config/integration.json
+      - ./docker/auth-service/default.json:/opt/auth-service/config/default.json
```

Comment:
You shouldn't need this if the NODE_ENV is `integration`, right?

Reply:
yes, it shouldn't this was my desperation attempt in trying to figure out why the wait4 script wasn't working and thinking that the mysql check might default to `default.json` database's config.

in reality `auth-service` has that old config setup that is not standardized with other services. another todo...

I'll remove it once the dependency images are built and I replace the github images

Just wanted to get some eyes on the PR before bed.

Reply:
Yep no worries.